retrieverdash/requirements.txt

```diff
@@ -1,4 +1,4 @@
-Django>=2.0.10
+Django==2.0.10
```

Comment:
why are we using ==?

Reply:
To fix the django version to 2.0.10.

Reply:
If we use `>=` it will start using Django 2.2. It's always better to pin the dependencies so that the project does not break.